duckduckgo/macos-browser
# Update Apperance Settings icon

Task/Issue URL: https://app.asana.com/0/1177771139624306/1203386494063701/f

**Description**:
Updating with an icon that renders correctly on macOS Ventura.



**Steps to test this PR**:
1. Go to Settings
1. Verify that Appearance settings icon shows a color wheel, and not a grey circle

**Testing checklist**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
